cobalt/browser/system_platform_error_handler.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The Cobalt Authors. All Rights Reserved.
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
```

Comment:
Copyright years don't need to be changed. Let's back those header changes out.

Reply:
Done

Reply:
Done